### PR TITLE
Release v12.1.0-beta

### PR DESCRIPTION
**Pull request by vladbarosan**
Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we can.

As part of your submission, please make sure that you can make the following assertions:

 - [ ] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
 - [ ] I've tested my changes, adding unit tests where applicable.
 - [ ] I've added Apache 2.0 Headers to the top of any new source files.
 - [ ] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
 - [x] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
 - [ ] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.

### CODE REVIEW

**Comment by vladbarosan** [comment]
@marstr if you can have a look

File: services/cognitiveservices/v1.0/computervision/models.go [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+
+const (
+	// En ...
+	En Language1 = "en"
+	// Zh ...
+	Zh Language1 = "zh"
+)
+
+// OcrLanguages enumerates the values for ocr languages.
+type OcrLanguages string
```
Comment (mcardosos) [inline services/cognitiveservices/v1.0/computervision/models.go]: Whats is the difference between this and `Language1`?

Comment (vladbarosan) [reply services/cognitiveservices/v1.0/computervision/models.go]: @mcardosos They come from 2 different enums in the Swagger specification for computer vision . `Language` is an enum for the `analyze` (`AnalyzeImage` in Go SDK ).
 
`OcrLanguage` is for the `ocr` (`RecognizerPrintedText` in Go SDK )

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## `v12.1.0-beta`
```
Comment (mcardosos) [inline CHANGELOG.md]: Should new API versions be included?

Comment (vladbarosan) [reply CHANGELOG.md]: We talked and we prefer not to include every new api version (as you can see in the v12 release), just to anounce the new services.

Comment (vladbarosan) [reply CHANGELOG.md]: but created an issue if we want to do that, to do it automatically https://github.com/Azure/azure-sdk-for-go/issues/931

Comment (mcardosos) [reply CHANGELOG.md]: Good :) I guess if we included new API versions it would be overkill

**Pull request merged**